### PR TITLE
Add two payments package for django-oscar

### PR DESCRIPTION
**Pull request by SalahAdDin**
Isn't easy maintain this packages without community help.
 In some countries isn't easy use paypal for payment, instead, is easy pay in moment, i mean, cash on delivery. And, for example in my country, PayU offers multiply payment methods: user credit card, debit card, cash on delivery, etc.

I think that many people find useful use this packages, and i'll like the Oscar's Official Team help us with this packages.

### CODE REVIEW

**Comment by mvantellingen** [comment]
hi @SalahAdDin, sorry for not merging earlier. Are you able to fix the merging conflicts?

File: README.rst [hunk 1]
```diff
@@ -180,6 +182,8 @@ Let us know if you're writing a new one!
 .. _django-oscar-adyen: https://github.com/oscaro/django-oscar-adyen
 .. _django-oscar-przelewy24: https://github.com/kisiel/django-oscar-przelewy24
 .. _oscar-sagepay: https://github.com/udox/oscar-sagepay
+.. _django-oscar-payu: https://github.com/SalahAdDin/django-oscar-payu
```
Comment (sasha0) [inline README.rst]: Is it fork of the https://bitbucket.org/spliceglobal/django-oscar-payu?

Comment (SalahAdDin) [inline README.rst]: Yes!

**Comment by sasha0** [comment]
PR needs to be rebased against upstream master due to the conflicts.

**Comment by SalahAdDin** [comment]
Ok, i will do it this night.

**Comment by sasha0** [comment]
PR still has conflicts. Feel free to rebase and submit new PR. Thanks.

**Comment by SalahAdDin** [comment]
@saha0 Which conflicts?

File: README.rst (hunk 1 above)
Comment (SalahAdDin) [reply README.rst]: Have i to put the original repository instead?

**Comment by polow** [comment]
An update to cash on delivery is much appreciated @SalahAdDin

**Comment by SalahAdDin** [comment]
> An update to cash on delivery is much appreciated @SalahAdDin

Sorry, I don't work with  **Django** since 2018.

**Pull request closed without merging**